pkgs/applications/networking/instant-messengers/bitlbee-discord/default.nix

```diff
@@ -2,13 +2,14 @@
 
 with stdenv.lib;
 stdenv.mkDerivation rec {
-  name = "bitlbee-discord-2017-12-27";
+  name = "bitlbee-discord-${version}";
```

Comment:
I recently stopped using bitlbee, in you are interested in skype as well: https://github.com/Mic92/nur-packages/blob/master/pkgs/purple-skypeweb/default.nix#L9